setup.py

```diff
@@ -19,10 +19,10 @@
 ]
 
 setup(
-    name='facerec',
+    name=config.name,
     version=config.__version__,
-    license='BSD',
-    url='http://github.com/rushabh-v/linux_face_unlock',
+    license=config.license,
+    url=config.url,
     description="A face Authentication system for Ubuntu Linux.",
```

Comment:
Just Capitalise all the variable names (except `__version__`) in the config and here as well. Rename `url` to `GIT_URL`